pkg/liqoctl/install/aks/aks_test.go

```diff
@@ -53,7 +53,7 @@ var _ = Describe("Extract elements from AKS", func() {
 		cmd := &cobra.Command{}
 
 		GenerateFlags(cmd)
-		cmd.Flags().String("cluster-name", "", "Name to assign to the Liqo Cluster")
+		cmd.Flags().String("cluster-name", "test-cluster", "Name to assign to the Liqo Cluster")
```

Comment:
we should mark this flag as required and not set a default value (the same for the other providers)

Reply:
This is where values are passed in tests. But you're right, looking at the surrounding code I should have used

```go
Expect(flags.Set("cluster-name", "test-cluster")).To(Succeed())
```

Reply:
It is required by ValidateGenericCommandArguments in pkg/liqoctl/install/provider/generic.go, and it must be declared in each test because it is not a custom flag (declared by the provider). However, I will split the declaration and the assignation of the value.